webapp/components/at_mention/at_mention.jsx

```diff
@@ -79,13 +80,15 @@ export default class AtMention extends React.PureComponent {
                     placement='right'
                     rootClose={true}
                     overlay={
-                        <ProfilePopover
-                            user={user}
-                            src={Client4.getProfilePictureUrl(user.id, user.last_picture_update)}
-                            hide={this.hideProfilePopover}
-                            isRHS={this.props.isRHS}
-                            hasMention={this.props.hasMention}
-                        />
+                        <Pluggable>
+                            <ProfilePopover
+                                user={user}
+                                src={Client4.getProfilePictureUrl(user.id, user.last_picture_update)}
+                                hide={this.hideProfilePopover}
+                                isRHS={this.props.isRHS}
+                                hasMention={this.props.hasMention}
+                            />
+                        </Pluggable>
```

Comment:
Probably just name the plugable versions their real names and have some prefix for the implementations. So you would have `ProfilePopover` be the plugable one and `ProfilePopoverImpl` be the non-pluggable implementation. Seems kinda messy though. Maybe it actually is better to just wrap it everywhere.